### PR TITLE
[SYCLomatic] [DOC] Update exsisting warning messages (align between code and doc)

### PR DESCRIPTION
**Pull request by zhiweij1**
Signed-off-by: Jiang, Zhiwei <zhiwei.jiang@intel.com>

### CODE REVIEW

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Comment by zhimingwang36** [comment]
@krisdale , please review this PR.

File: docs/dev_guide/diagnostic_ref/dpct1013.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ Message
 
 The rounding mode could not be specified and the generated code may have different
 accuracy than the original code. Verify the correctness. SYCL math built-in
-rounding mode is aligned with OpenCL C 1.2 standard.
+functions rounding mode is aligned with OpenCL C 1.2 standard.
```
Comment (krisdale) [inline docs/dev_guide/diagnostic_ref/dpct1013.rst]: Update to "function" instead of functions.

File: docs/dev_guide/diagnostic_ref/dpct1063.rst [hunk 2]
```diff
@@ -8,7 +8,7 @@ Message
 
 .. _msg-1063-start:
 
-Advice parameter is device-defined. You may need to adjust it.
+Advice parameter is device-defined``<addtional action>``. You may need to adjust it.
```
Comment (krisdale) [inline docs/dev_guide/diagnostic_ref/dpct1063.rst]: Spelling error and spacing issue: Add space after device-defined and update spelling to "additional"

**Review by krisdale — CHANGES_REQUESTED** [review]
A couple minor updates needed. Approved once the updates are made as I described.

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**